admin/class-bln-publisher-admin.php

```diff
@@ -175,5 +175,18 @@ function render_webln_donation_button($attributes)
             </div>';
     }
 
+    /**
+     * Add a Bitcoin Lightning address field to WordPress user profile page
+     * 
+     * @param  array $methods
+     * @return array
+     *
+     * @link https://developer.wordpress.org/reference/hooks/user_contactmethods/
+     */
+    function add_user_lnp_address( $methods )
+    {
+        $methods['_lnurl_meta_tag_lnurlp'] = __('Ligtning Address', 'lnp-alby');
```

Comment:
I would name this something global that we can use it in other cases, too. 
maybe just `$methods['_ln_address']` ?

Reply:
Yes, ok.
In general, it's wise to prefix database fields to reduce possiblity of some else plugin having the same field name.
These fields are saved as `key => value`, I would suggest add `_lnp_ln_address` (we already prefix options with lnp_ too)